articles/communication-services/concepts/known-issues.md

```diff
@@ -121,3 +122,17 @@ Developers are encouraged to do the following:
 
 #### Possible causes
 In some browsers (Safari for example), acquiring your own stream from the same device will have a side-effect of running into race conditions. Acquiring streams from other devices may lead the user into insufficient USB/IO bandwidth, and sourceUnavailableError rate will skyrocket.  
+
+
+# Azure Communication Services Call Automation APIs
+
+Following are the known issues in the Azure Communication Services Call Automation APIs
+
+1. The only authentication supported at this time for server applications is using a connection string.
```

Comment:
since these aren't in a specific order, they should be an un-ordered list rather than an ordered list.